### PR TITLE
turn-howto.md: enhance and add Let's Encrypt instructions

### PR DESCRIPTION
**Pull request by rubo77**
This will make it obvious, that you have to use the key defined in your TURN-Server and add a Lets Encrypt example

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

File: docs/turn-howto.md [hunk 1]
```diff
@@ -19,73 +19,94 @@ The TURN daemon `coturn` is available from a variety of sources such as native p
...
 #### Debian installation
 
-    # apt install coturn
+1. install the debian package
+
+     apt install coturn
+
+2. install the systemd service
+
+      wget https://raw.githubusercontent.com/coturn/coturn/master/examples/etc/coturn.service -O /etc/systemd/system/coturn.service
```
Comment (richvdh) [inline docs/turn-howto.md:28]: does the apt package really not include a systemd unit file?

File: changelog.d/7232.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Add systemd service, Let's Encrypt instructions and also clean up markdown, clarify the key usage and remove binary path.
```
Comment (richvdh) [inline changelog.d/7232.doc:1]: this is a bit misleading. synapse already has all these things.

why don't you just say

```suggestion
Update `turn-howto.md`. Contributed by @rubo77.
```

File: docs/turn-howto.md (hunk 1 above)
Comment (rubo77) [reply docs/turn-howto.md:28]: yes, really not ;)

Comment (richvdh) [reply docs/turn-howto.md:32]: If I download it and find one, I am going to send you an invoice for my time. Agreed?

Comment (rubo77) [reply docs/turn-howto.md:32]: wait... :)

Comment (rubo77) [reply docs/turn-howto.md:32]: strange:

```
# apt-file search coturn|grep servi
coturn: /lib/systemd/system/coturn.service
```
I don't understand, why i didnt find it initially, maybe the service is called `turnserver.service` instead?

Comment (rubo77) [reply docs/turn-howto.md]: I removed the wget command

**Comment by rubo77** [comment]
> also: have you actually looked at the rendered output?

Oh, there was an indendtion for each the command lines missing

fixed

File: docs/turn-howto.md [hunk 3]
```diff
@@ -19,13 +19,25 @@ The TURN daemon `coturn` is available from a variety of sources such as native p
...
 
-    # apt install coturn
+1. install the debian package
+
+        apt install coturn
+
+2. prepare sqlite-DB vor coturn
+
+        mkdir -p /var/lib/turn/
+        chown turnserver:turnserver /var/lib/turn/
```
[the remark this answers is not in the document]
Comment (richvdh) [reply docs/turn-howto.md:29]: It seems fine without this in my testing.

**Comment by rubo77** [comment]
Why is this closed? Are some of the edited suggestions added?

**Comment by richvdh** [comment]
As I said, I've taken the bits that seemed to be correct and used them to write #7533, which has been merged. If you think I've missed anything, please could you say what they were?

**Pull request closed without merging**